src/Components/QuestionList/QuestionList.tsx

```diff
@@ -23,40 +29,24 @@ export const QuestionList = ({ setShowQuestionList }: any) => {
         type: "Js",
         name: "",
         hideCorrect: false,
-        difficulty: "null",
+        difficulty: "" as Difficulty,
```

Comment:
`""` is not a valid Difficulty. You need to either default this to `Difficulty.Easy` or create a new item like `Difficulty.None` for the default cause. Up to you.